### PR TITLE
cleanup puppet-strings documentation

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -69,4 +69,23 @@ begin
...
+    rocky = {'operatingsystem' => 'Rocky', 'operatingsystemrelease' => versions}
+    metadata['operatingsystem_support'] << alma unless metadata['operatingsystem_support'].any?{|operatingsystem| operatingsystem['operatingsystem'] == 'AlmaLinux' }
+    metadata['operatingsystem_support'] << rocky unless metadata['operatingsystem_support'].any?{|operatingsystem| operatingsystem['operatingsystem'] == 'Rocky' }
+    puts metadata
+    File.open('metadata.json', 'w') do |f|
+      f.write(JSON.pretty_generate(metadata) + "\n")
+    end
+  end
+end
+
```
Comment (kenyon) [inline Rakefile:90]: Is this supposed to be in this change? Seems unrelated to docs cleanup.

Comment (bastelfreak) [reply Rakefile:90]: ah this wasn't supposed to be in the PR :D will clean it up

**Review by kenyon — APPROVED** [review]

**Review by smortex — APPROVED** [review]
